test/helpers/deployer.js

```diff
@@ -412,10 +418,15 @@ class Deployer {
 
   async initializeChildChain(owner, options = { updateRegistry: true }) {
     // not mentioning the gas limit fails with "The contract code couldn't be stored, please check your gas limit." intermittently which is super weird
-    this.childChain = await contracts.ChildChain.new({ gas: 7500000 })
+    this.childChain = await contracts.ChildChain.new({ gas: 75000000 })
     await this.childChain.changeStateSyncerAddress(owner)
     if (!this.globalMatic) {
       // MRC20 comes as a genesis-contract at utils.ChildMaticTokenAddress
+      if (!utils.ChildMaticTokenAddress) {
```

Comment:
Please explain this if block? `utils.ChildMaticTokenAddress` is set right?

Reply:
Yes, it's a quick solution to re-deploy matic native token on the ganache instead during coverage. Will be removed when multiple providers will be supported.